aviary/docs/getting_started/installation.md

```diff
@@ -2,16 +2,33 @@
 
 ## Quick start installation
 
-The simplest installation method for development is an "editable mode" install with ``pip``.
-From within the base folder of Aviary, perform this command in your terminal:
+The simplest installation method for users is to install via pip.
+Once you have cloned the Aviary repo, change directories into the top-level Aviary folder (not within the `aviary` folder) and run the following command:
+
+    pip install .
+
+If you want to also run the Aviary test suite (this is not strictly necessary), you can instead run:
+
+    pip install .[test]
+
+If you also want to install all packages used for the Aviary tests _and_ external subsystem examples, you can instead run:
+
+    pip install .[all]
+
+If you are a developer and plan to modify parts of the Aviary code, install in an "editable mode" with ``pip``:
```

Comment:
Is there a reason pip is ``pip`` here but just pip on line 5?

Reply:
Nope, changed!